uber/config.py

```diff
@@ -126,6 +126,10 @@ def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
     @property
```

Comment:
This seemed generically useful enough to put here instead of special-casing the one page where I currently need it.